regression/jbmc-strings/java_char_array_init/test.desc

```diff
@@ -8,3 +8,9 @@ assertion.* file test_init.java line 33 .* SUCCESS$
 assertion.* file test_init.java line 35 .* FAILURE$
 assertion.* file test_init.java line 37 .* FAILURE$
 --
+--
+Runs indefinitely under symex-driven lazy loading. Unsure why yet; this could
+be a bug, or could simply be variations in order of VCCs / functions perturbing
+the string solver and leading to a long unfruitful search that would actually
+terminate eventually.
```

Comment:
Please debug properly, for example by comparing the results of `--program-only`.

Reply:
These two have now been solved and therefore this change dropped.